### PR TITLE
feat: add remove/removeSync for fs modules

### PR DESCRIPTION
**Pull request by axetroy**
part of #261

### CODE REVIEW

File: fs/remove.ts [hunk 1]
```diff
@@ -0,0 +1,17 @@
...
+// It should have its own interface in case there will be new option field in the future
+/* eslint-disable-next-line @typescript-eslint/no-empty-interface */
+export interface RemoveOptions extends Deno.RemoveOption {}
+
+/* Removes the named file or directory. alias for `Deno.remove` */
+export async function remove(
+  path: string,
+  options: RemoveOptions = {}
+): Promise<void> {
+  await Deno.remove(path, options);
```
Comment (ry) [inline fs/remove.ts]: This is just an alias?

Comment (axetroy) [reply fs/remove.ts]: yes. see here https://github.com/denoland/deno_std/issues/261#issuecomment-472533659

Comment (axetroy) [reply fs/remove.ts]: @ry
I have re-consider the remove function.

It should accept an argument for `target path`.

make sure it must remove success whatever target is a `file` or `folder` or `link`.

This is different from `Deno.remove`, do you think it is feasible?

Comment (ry) [reply fs/remove.ts]: Can you explain in the documentation in detail how this is different than Deno.remove? 

I also think it should have a different name. Something like "removeAnything" (but that's not very good, please suggest a better name)

@piscisaureus @kevinkassimo Do you have opinions on this?

Comment (kevinkassimo) [reply fs/remove.ts]: I'm mostly okay with the aliasing idea, but still preferring an explicit `recursive` option...

Comment (ry) [reply fs/remove.ts]: @axetroy For a file, folder and a link, can you please list out the current API to delete each of these?

Comment (axetroy) [reply fs/remove.ts]: @ry

- [x] Remove `file` by using `Deno.remove()`
- [x] Remove `folder` by using `Deno.remove({ recursive: true })`
- [ ] Remove `symlink` by using `Deno.unlink()` **Not implement in Deno core**

Comment (ry) [reply fs/remove.ts]: @axetroy So let's just make Deno.remove() work for symlinks instead of this PR.

Comment (axetroy) [reply fs/remove.ts]: @ry ok. I will close this soon if there are no more questions about this PR.

Comment (ry) [reply fs/remove.ts]: Yes, close this one. Can you submit the tests to https://github.com/denoland/deno/blob/master/js/remove_test.ts
The implementation should probably be in Rust here:
https://github.com/denoland/deno/blob/e43da28b28c8b1ed97da603ec0e6bf91fbb67798/cli/ops.rs#L1025-L1052

**Pull request closed without merging**